spec/helpers/sign_up_helper_spec.rb

```diff
@@ -7,3 +7,12 @@ def sign_up
   fill_in('user_password', with: 'secret')
   click_button('Sign up')
 end
+
+def sign_up_other
```

Comment:
To keep things DRY we probably ought not repeat this method; 

We can let the existing sign_up helper take a parameter that allows you to give an alternative email address for the user signing up - but also make it optional so you would only have to give the email when you wanted to use one different to a default email.